org.scala-ide.sdt.core/plugin.xml

```diff
@@ -1838,6 +1838,42 @@
           The font used for all Scala hovers.
        </description>
     </fontDefinition>
+    <colorDefinition
+          categoryId="org.scalaide.ui.theme"
+          id="org.scalaide.ui.color.interpreterForeground"
+          label="Interpreter foreground color"
+          value="60,0,128">
```

Comment:
I didn't understand he difference between `defaultTo` and `value` when I read their description, now I don't understand what you mean by "colour in the Eclipse platform". I think the ColorThemes plugin just changes the values that belong to all existing preferences and therefore this plugin would also need a patch to support these values. Or do you mean that we should not use our own ids?